### PR TITLE
Product-cards

### PR DESCRIPTION
**Pull request by DmitryVolchokFE**
https://dmitryvolchokfe.github.io/layout_product-cards/
https://dmitryvolchokfe.github.io/layout_product-cards/report/html_report/

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -1,13 +1,83 @@
...
+        <div class="product-card_picture-position">
+          <img
+          class="product-picture"
+          src="./images/imac.jpeg"
+          alt="imac"
+        >
+        </div>
+
+        <div class="product-card__text-block">
+          <span class="product-card__name">
```
Comment (vlad-demchuk) [inline src/index.html:31]: it will be better to use `h2, h3 ...` tag for titles

File: src/index.html [hunk 2]
```diff
@@ -1,13 +1,83 @@
...
-    <h1>Product cards</h1>
+  <!-- #endregion -->
+  <body class="page">
+    <header>
+    </header>
+
+    <main>
+      <!-- #region product-card -->
+      <div class="product-card" data-qa="card">
+        <div class="product-card_picture-position">
```
Comment (vlad-demchuk) [inline src/index.html:22]: it doesn't correspond to bem naming convention
```
<div class="product-card_picture-position">

File: src/index.html [hunk 3]
```diff
@@ -1,13 +1,83 @@
...
+          <img
+          class="product-picture"
+          src="./images/imac.jpeg"
+          alt="imac"
+        >
```
Comment (vlad-demchuk) [inline src/index.html:27]:
```suggestion
        <img
          class="product-picture"
          src="./images/imac.jpeg"
          alt="imac"
        >
```

Comment (vlad-demchuk) [reply src/index.html:27]: check your code style

File: src/index.html [hunk 4]
```diff
@@ -1,13 +1,83 @@
...
+          <span class="product-card__name">
+            APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+          </span>
+
+          <span class="product-card__code">
+            Product code: 195434
+          </span>
+
+          <div class="product-card__feedback-block">
+            <div class="product-card__stars">
```
Comment (vlad-demchuk) [inline src/index.html:40]: task requirement
image

File: src/index.html [hunk 5]
```diff
@@ -1,13 +1,83 @@
...
+            <span class="product-card__price-text">
+              Price:
+            </span>
+
+            <span class="product-card__price-numbers">
+              $2,199
+            </span>
+          </div>
+          
+          <div>
```
Comment (vlad-demchuk) [inline src/index.html:63]: redundant `div`

File: src/styles/blocks/product-card.css [hunk 6]
```diff
@@ -0,0 +1,130 @@
+/* #region product-card */
```
Comment (vlad-demchuk) [inline src/styles/blocks/product-card.css:1]: no need to leave comments, regions

File: src/styles/blocks/product-card.css [hunk 7]
```diff
@@ -0,0 +1,130 @@
...
+  border-radius: 5px;
+}
+
+.product-picture {
+  width: 160px;
+  height: 134px;
+}
+
+.product-card_picture-position {
+  margin: 32px 19px 0;
```
Comment (vlad-demchuk) [inline src/styles/blocks/product-card.css:19]: you can use `auto` for horizontal margins to center the image

File: src/styles/blocks/product-card.css [hunk 8]
```diff
@@ -0,0 +1,130 @@
...
+}
+
+.product-card__code {
+  display: inline-block;
+  font-size: 10px;
+  font-weight: 400;
+  line-height: 14px;
+  color: #616070;
+  margin-bottom: 16px;
+  margin-top: 4px;
```
Comment (vlad-demchuk) [inline src/styles/blocks/product-card.css:39]: image
checklist

File: src/styles/blocks/product-card.css [hunk 9]
```diff
@@ -0,0 +1,130 @@
...
+.product-card__star {
+  background-image: url(../../images/star.svg);
+  width: 16px;
+  height: 16px;
+  background-repeat: no-repeat;
+  background-position: center;
+  margin-right: 4px;
+}
+
+.product-card__star:nth-child(-n + 4) {
```
Comment (vlad-demchuk) [inline src/styles/blocks/product-card.css:57]: it should work not only with 4 stars 

use styles from the previous task

File: src/styles/blocks/product-card.css [hunk 10]
```diff
@@ -0,0 +1,130 @@
...
+  justify-content: space-between;
+  align-items: flex-end;
+  margin-bottom: 24px;
+}
+
+.product-card__feedback-numbers {
+  font-size: 10px;
+  font-weight: 400;
+  line-height: 14px;
+  color: #060b35;
```
Comment (vlad-demchuk) [inline src/styles/blocks/product-card.css:73]: use variables for repeatable values like colors

File: src/styles/style.css [hunk 11]
```diff
@@ -0,0 +1,2 @@
+@import url(C:/Users/Zveroboy/DmitryVolchok/projects/hello-world/layout_product-cards/src/styles/blocks/page.css);
+@import url(C:/Users/Zveroboy/DmitryVolchok/projects/hello-world/layout_product-cards/src/styles/blocks/product-card.css);
```
Comment (vlad-demchuk) [inline src/styles/style.css:2]: use relative paths

**Review by vlad-demchuk — CHANGES_REQUESTED** [review]

File: src/styles/blocks/product-card.css [hunk 12]
```diff
@@ -0,0 +1,96 @@
+/* #region card */
```
Comment (witflash) [inline src/styles/blocks/product-card.css:1]: There is no need to use such comments. You can remove them all

File: src/styles/blocks/product-card.css
Comment (witflash) [inline src/styles/blocks/product-card.css:1]: If your block is named `card` it's better to call the file with the same name: `blocks/card.css`

**Review by witflash — APPROVED** [review]
Good work 👍🏼

**Pull request closed without merging**